Makefile

```diff
@@ -1,7 +1,7 @@
 install: composer assets
 
 composer:
-	composer install
+	composer install --ansi --prefer-dist --no-interaction --no-progress --classmap-authoritative
```

Comment:
It is maybe ok for CI, but I'm afraid we can't do it here:
https://github.com/PrestaShop/PrestaShop/pull/22146
https://github.com/PrestaShop/PrestaShop/issues/22141 :sweat_smile: 
We also need to keep the progress and remove ansi for CI.

Reply:
I will revert the changes on `Makefile`.

About ansi/progress, I did not introduce it, see https://github.com/PrestaShop/PrestaShop/pull/23964/files#diff-a73bb6555480a5ee79ae276a3f5d71a08fa316e09a4a8da7b643cf1e92c97df9R29 , I only normalized it across other uses. Please advise what to do.